testcases/PowerNVDump.py

```diff
@@ -132,13 +142,38 @@ def setup_test(self, dump_place="local"):
         if self.distro == "rhel":
             self.c.run_command("cp /etc/kdump.conf_bck /etc/kdump.conf")
         if self.distro == "sles":
-            self.c.run_command("cp /etc/sysconfig/kdump_bck /etc/sysconfig/kdump")
+            self.c.run_command(
+                "cp /etc/sysconfig/kdump_bck /etc/sysconfig/kdump")
         if dump_place == "local":
             self.crash_content = self.c.run_command(
                 "ls -l /var/crash | grep '^d'| awk '{print $9}'")
         if dump_place == "net":
             self.crash_content = self.c.run_command(
-                "ssh root@%s \"ls -l %s | grep '^d'\" | awk '{print $9}'" % (self.dump_server_ip, self.dump_path))
+                "ssh -i %s root@%s \"ls -l %s | grep '^d'\" | awk '{print $9}'" % (self.rsa_path, self.dump_server_ip, self.dump_path))
+            log.debug("crash content is %s" % (self.crash_content))
+
+    def setup_pwdless_auth(self):
+        # Setup Pulickey authentication for Kdump over ssh and NFS
+        # creates key using ssh-keygen and copies key using sshpass
+        # check for the successful ssh connection
+        self.c.run_command("rm -f %s" % self.rsa_path)
+        if self.distro == "rhel":
+            self.c.run_command("yum -y install sshpass")
+        elif self.distro == "sles":
+            self.c.run_command("zypper install -y sshpass")
+        else:
+            self.c.run_command("apt-get install -y sshpass")
```

Comment:
We need a check to validate the package installation actually worked. What will happen if the software repositories are not configured and installing the sshpass command fails ?